zookeeper/core/component.py

```diff
@@ -289,14 +289,20 @@ def _list_field_strings(instance, color: bool, single_line: bool) -> Iterator[st
         except AttributeError as e:
             if field.allow_missing:
                 value = utils.missing
+                yield f"{field_name}={value}" if color else f"{field_name}={value}"
+                continue
             else:
                 raise e from None
 
         parent_instance = next(
             utils.generate_component_ancestors_with_field(instance, field_name), None
         )
         if parent_instance is not None:
-            is_inherited = base_getattr(parent_instance, field_name) == value  # type: ignore
+            parent_value = base_getattr(parent_instance, field_name)  # type: ignore
+            if is_tensor(parent_value):
+                is_inherited = parent_value is value
+            else:
+                is_inherited = parent_value == value
```

Comment:
@AdamHillier  I'm not sure if we even need to make this distinction; if it really is an inherited value I think we could get away with `parent_value is value` in any case?